.github/workflows/claude-sign.yml

```
@@ -0,0 +1,135 @@
+          - dry-run
+          - sign-tip-only
+          - sign-all-unsigned
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  sign:
+    runs-on: ubuntu-latest
+    if: env.OP_SERVICE_ACCOUNT_TOKEN != ''
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)